Sources/Optimizely/OptimizelyClient+ObjC.swift

```diff
@@ -304,6 +304,20 @@ extension OptimizelyClient {
         try self.track(eventKey: eventKey, userId: userId, attributes: attributes, eventTags: eventTags)
     }
     
+    /// Read a copy of project configuration data model.
+    ///
+    /// This call returns a snapshot of the current project configuration.
+    /// If a datafile change is notified (NotificationType.datafileChange), this method should be called again to get the updated configuration data.
+    ///
+    /// - Returns: a snapshot of public project configuration data model
+    /// - Throws: `OptimizelyError` if SDK is not ready
+    @available(swift, obsoleted: 1.0)
+    @objc(getOptimizelyConfigWithError:)
```

Comment:
I guess you're asking if we can use property for "optimizelyConfig" instead of get function. It's a possible option, but we can integrate error throw/return and also in this way, more consistent with other APIs.